docs/linting.md

```diff
@@ -0,0 +1,72 @@
+# Linting
+
+## Installation
+
+You must declare your linters as Bazel aspects.
+
+> This is needed because Bazel only allows aspect attributes of type
+> `bool`, `int` or `string`.
+> We want to accept label-typed attributes, so we follow the documentation from
+> https://bazel.build/extending/aspects#aspect_definition_2
+
+We suggest creating a `lint.bzl` file in whatever package contains most of your
+custom Bazel configuration, commonly in `tools/`.
+This `lint.bzl` should contain linter aspect declarations.
+See the `docs/` folder for "aspect factory functions" that declare your linters.
+
+Finally, reference those linters as `//path/to:lint.bzl%mylinter`
+in the lint runner.
+
+If you use the Aspect CLI, then include a block like the following in `.aspect/cli/config.yaml`:
+
+```yaml
+plugins:
+  - name: lint-plugin
```

Comment:
missing `from:` ?

Reply:
yeah... but we haven't published the plugin on a release yet, so there's not a good `from` value to document...

Reply:
it will eventually just be `from: rules_lint` because we default to github.com/aspect-build